core/src/main/java/io/kestra/core/secret/SecretService.java

```diff
@@ -17,10 +19,20 @@ public class SecretService {
 
     @PostConstruct
     private void postConstruct() {
-        decodedSecrets = System.getenv().entrySet().stream().filter(entry -> entry.getKey().startsWith(SECRET_PREFIX)).collect(Collectors.toMap(
-            entry -> entry.getKey().substring(SECRET_PREFIX.length()).toUpperCase(),
-            entry -> new String(Base64.getDecoder().decode(entry.getValue()))
-        ));
+        Logger logger = LoggerFactory.getLogger(SecretService.class);
```

Comment:
We usually use `@Slf4j` at the class level.
At least add it as a constant it's a best practice even if only used there.